cli/src/main/java/com/devonfw/tools/ide/tool/jasypt/JasyptUrlUpdater.java

```diff
@@ -24,4 +32,11 @@ protected String getMavenArtifcatId() {
     return "jasypt";
   }
 
+  @Override
+  public boolean isValidVersion(String version) {
+
+    DefaultArtifactVersion artifactVersion = new DefaultArtifactVersion(version);
+    return artifactVersion.compareTo(MIN_VERSION) >= 0;
```

Comment:
Could you just use `VersionIdentifier` instead of `DefaultArtifactVersion` to avoid the overhead of maven artifact dependency?
`VersionIdentifier` also has Methods like `isGreaterOrEqual` that will make the code more readable.

Reply:
implemented!